src/SWP/Bundle/WebRendererBundle/Resources/config/services.yml

```diff
@@ -33,6 +33,17 @@ services:
         class: '%swp_webrenderer.twig_extension.class%'
         public: false
         arguments:
-            - '@swp_webrenderer.theme.context.tenant_aware'
+            - "@swp_webrenderer.theme.context.tenant_aware"
         tags:
             - { name: twig.extension }
+
+    swp_webrenderer.theme.resolver.template_name:
+        class: SWP\Bundle\WebRendererBundle\Resolver\TemplateNameResolver
```

Comment:
lets move it the the `parameters:` section in this file, will be easier to override the class, without need to override the whole service definition, if needed in the future